BudgetTime.java

```diff
@@ -83,12 +162,36 @@ private int timeInput() throws IOException {
 		return 0;
 	}
 
-	public static void main(String[] args) {
+	/**
+	 * Called from the main program to give user budget options
+	 * Handles all IOExceptions thrown at other points in class
+	 */
+	public void mainMenu() {

```

Comment:
should we loop this, incase they want to ammend multiple? would stop them returning to main menu then having to go back into this menu.

Reply:
Yep, good idea. Will implement this now